jekyll/_cci2/server/installation/migrate-from-server-3-to-server-4.adoc

```diff
@@ -0,0 +1,209 @@
+---
+version:
+- Server v4.x
+- Server Admin
+---
+= Migrate from server v3.x to v4.x
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Learn now to migrate from CircleCI server v3.x to v4.x.
+:icons: font
+:toc: macro
+:toc-title:
+
+Migrating from server v3.x to v4.x is and _in-place_ migration. You will generate a helm-value file, which will be used in a helm command to upgrade the CircleCI server from v3.x to v4.x.
+
+We recommend using a staging environment before completing this process in a production environment. This will allow you to gain a better understanding of the migration process, and give you an idea of how long the migration will take to complete.
```

Comment:
This line doesn't take into consideration how long the workflows migration will take to run in a prod environment

Reply:
@ryanwohara do you think I need to add a note that they need to add on x amount of time for workflows migration, or shall I just remove that part and leave it at they will better understand the migration process?

Reply:
It's ok I think @soulchips has added info about this in another PR